operator-framework/src/main/java/io/javaoperatorsdk/operator/api/RetryInfo.java

```diff
@@ -2,19 +2,24 @@
 
 public class RetryInfo {
 
-  private int retryNumber;
+  private int attemptIndex;
   private boolean lastAttempt;
 
   public RetryInfo(int retryNumber, boolean lastAttempt) {
-    this.retryNumber = retryNumber;
+    this.attemptIndex = retryNumber;
     this.lastAttempt = lastAttempt;
   }
 
-  public int getRetryNumber() {
-    return retryNumber;
+  public int getAttemptIndex() {
```

Comment:
Wouldn't `attemptCount` be a better name?

Reply:
Hmm, fine by me, I meant with this that the current execution attempt index is this number.